Jenkinsfile

```diff
@@ -2,5 +2,5 @@
 
 node {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-  govuk.buildProject()
+  govuk.buildProject(sassLint: false)
```

Comment:
It was enabled as a new default option in CI, but failed catastrophically when it ran against this codebase. Think there was an error in selecting for vendored content, and that may have been fixed now.